asserts/asserts.go

```diff
@@ -1021,6 +1022,16 @@ func assemble(headers map[string]interface{}, body, content, signature []byte) (
 		return nil, fmt.Errorf("unknown assertion type: %q", typ)
 	}
 
+	if assertType.Name == "aspect-bundle" {
+		if body == nil {
+			return nil, fmt.Errorf(`assertion aspect-bundle: body must contain aspect schema`)
+		}
+
+		if body, err = formatJSON(body); err != nil {
+			return nil, fmt.Errorf("assertion aspect-bundle: invalid schema: %w", err)
+		}
+	}
```

Comment:
we need to chat separately about this change in particular, it's unclear to me whether we should do this or instead just make sue that body and formatJSON(body) are the same?